front/app/components/ReactionControl/utils.ts

```diff
@@ -17,13 +13,22 @@ export const showIdeaReactions = (
   const likesCount = idea.attributes.likes_count;
   const dislikesCount = idea.attributes.dislikes_count;
   return (
-    participationMethod !== 'voting' &&
-    participationMethod !== 'proposals' &&
```

Comment:
We ensure the participation method is correct outside of this function for now, in a much more explicit manner. Perhaps a hook would be useful to check for the correct PM but keeping the scope limited with the new tandem cycle starting soon.